hr_holidays_leave_repeated/models/hr_leave.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2016-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class HrLeave(models.Model):
+    _inherit = "hr.leave"
+
+    repeat_every = fields.Selection([
+        ('workday', 'Every workday'),
+        ('week', 'Every week'),
+        ('biweek', 'Every two weeks'),
+        ('month', 'Every four weeks')
+    ])
+    repeat_mode = fields.Selection([
+        ('times', 'Number of Times'),
+        ('date', 'End Date'),
+    ], default='times')
+    holiday_type_repeat = fields.Boolean(related='holiday_status_id.repeat')
+    repeat_limit = fields.Integer(default=1, string='Repeat # times')
+    repeat_end_date = fields.Date(
+        'Repeat Until', default=lambda self: fields.Date.today())
```

Comment:
@astirpe 
Whe should choose for end date or repeat until throughout the module. I would make it easy and use end date.

Reply:
Just changing it to _End Date_ causes the following warning:
`Two fields (repeat_end_date, date_to) of hr.leave() have the same label: End Date.`

So I'm going to change it to: _Repeat End Date_.